Runtime/Scripts/Manager.cs

```diff
@@ -20,33 +20,43 @@ public class Manager : MonoBehaviour
         /// <summary>
         /// The port the project should use. 
         /// </summary>
+        [Tooltip("Server port for the MQTT-broker.")]
```

Comment:
Syns vi burde være konsekvente med å kalle det "MQTT server".

```suggestion
        [Tooltip("Server port for the MQTT server.")]
```